crowbar_framework/config/initializers/checks.rb

```diff
@@ -15,13 +14,10 @@
 # limitations under the License.
 #
 
-begin
-  Chef::Config.tap do |config|
-    config.node_name ENV["CHEF_NODE_NAME"]
-    config.client_key ENV["CHEF_CLIENT_KEY"]
-    config.chef_server_url ENV["CHEF_SERVER_URL"]
-    config.http_retry_count 3
+unless ENV["SKIP_CHECKS"] && ENV["SKIP_CHECKS"] == "yes"
+  network_checks = Crowbar::Checks::Network.new(Rails.logger)
+
+  unless network_checks.run_all_checks
+    raise "Crowbar network checks failed."
```

Comment:
It's temporary until we have the UI page, so I would keep it as simple as possible.